### PR TITLE
access_token route

### PR DESCRIPTION
**Pull request by cadecairos**

### CODE REVIEW

**Comment by cadecairos** [comment]
R? @jbuck

File: web/server.js [hunk 1]
```diff
@@ -102,8 +102,73 @@ module.exports = function(options) {
...
+                }
+                if ( client.client_secret !== request.payload.client_secret ) {
+                  return reply(Boom.forbidden('Invalid Client Credentials'));
+                }
+                reply(client);
+              });
+            }
+          },
+          {
+            assign: 'redirectUri',
```
Comment (jbuck) [inline web/server.js]: I was actually speccing out that we ignore any application-submitted redirect url entirely, and only use the registered one

File: web/server.js [hunk 2]
```diff
@@ -102,8 +102,73 @@ module.exports = function(options) {
...
+                Date.now() + (1000 * 60 * 60 * 24),
+                reply
+              );
+            }
+          }
+        ]
+      },
       handler: function(request, reply) {
-        reply('ok');
+        var redirectObj = url.parse(request.pre.redirectUri, true);
```
Comment (jbuck) [inline web/server.js]: so instead you could use `request.pre.client.redirect_uri`

**Comment by jbuck** [comment]
you can remove the variable redirect_uri bit. That'll simplify things a bit!

**Comment by jbuck** [comment]
@cadecairos r+

**Pull request merged**